src/net.h

```diff
@@ -978,6 +967,8 @@ class CNode
     }
 
 private:
+    friend class CConnman;
```

Comment:
I do not like the second commit a542bf10f0 `net: [refactor] move ReceiveMsgBytes to CConnman` - it reduces the encapsulation of `CNode` and goes against https://github.com/bitcoin/bitcoin/pull/27257. Its commit message says:

>    This allows ReceiveMsgBytes() to access CConnman members without needing
>    to copy them to every CNode. This is used in the next commit.

But I don't think the next commit b4d1007221 `net: Cache -capturemessages setting` does any of that, hmm?